src/LowercaseSentenceTokenizer.java

```diff
@@ -29,8 +30,18 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    * @return a list of tokens, where each token is a word or a period
    */
   public List<String> tokenize(Scanner scanner) {
-    // TODO: Implement this function to convert the scanner's input to a list of words and periods
-    return null;
+    List<String> words = new ArrayList<String>();
+    while(scanner.hasNext()){
+      String token = scanner.next();
+      if (token.charAt(token.length() - 1) == '.'){
+        words.add(token.substring(0, token.length() - 1));
+        words.add(".");
+      }
+      else {
+      words.add(token.toLowerCase());
+      }
```

Comment:
Nice, this works great! As an alternative, can you consider what this would look like using `endsWith` instead?